tsrelay/handler/get_serve.go

```diff
@@ -40,16 +40,22 @@ type currentTailnet struct {
 	MagicDNSEnabled bool
 }
 
+type selfStatus struct {
+	peerStatus
+	CurrentTailnet currentTailnet
+}
```

Comment:
`tailscale status --json` does not include a `CurrentTailnet` for peers. We could make it optional, but it shouldn't be optional inside Self. Do the Go client types suggest otherwise?